packages/storybook/src/spright/chat/chat-matrix.stories.ts

```diff
@@ -0,0 +1,238 @@
+
+You can also use the deployment utility to include the TestStand Engine and supporting files, LabVIEW and LabWindows/CVI Run-Time Engines, and hardware drivers in the installer you create. The installer that the deployment utility creates can also register ActiveX servers, replace existing files on the target computer, and create program shortcuts. You can configure the deployment utility to remove VI block diagrams or to lock the VIs you deploy.
+`;
+
+const component = ([
+    chatMessageTypeName,
+    messageType
+]: ChatMessageTypeState): ViewTemplate => html`
+
+    <${chatMessageTag}
+        message-type=${() => messageType}
+        style="margin-right: 8px;">
+            ${() => `Message Type: ${chatMessageTypeName}`}</${chatMessageTag}>
+`;
+
+const differentChatMessageSizeTestCase = (
+    [message]: [string],
+    [messageType]: [string]
+): ViewTemplate => html`
+    <div style="margin: 10px 0px;"> 
+        <${chatMessageTag}
+            message-type=${() => messageType}
+            style="outline: 1px dashed red;">
+            <${richTextViewerTag}
+                :markdown="${_ => message}">
+            </${richTextViewerTag}
+        </${chatMessageTag}>
+    </div>
+`;
+
+const differentChatConversationSizeTestCase = ([message]: [
+    string
+]): ViewTemplate => html`
+    <${chatConversationTag}
+        style="width: 900px; height: 400px;">
+        <${chatMessageTag} message-type='outbound' style="outline: 1px dashed red;">
+            <${richTextViewerTag} :markdown="${_x => message}"></${richTextViewerTag}>
+        </${chatMessageTag}>
+        <${chatMessageTag} message-type='inbound' style="outline: 1px dashed red;">
+            <${richTextViewerTag} :markdown="${_x => message}"></${richTextViewerTag}>
+        </${chatMessageTag}>
+        <${chatMessageTag} message-type='system' style="outline: 1px dashed red;">
+            <${richTextViewerTag} :markdown="${_x => message}"></${richTextViewerTag}>
+        </${chatMessageTag}>
+    </${chatConversationTag}>
+`;
+
+export const chatMessageThemeMatrix: StoryFn = createMatrixThemeStory(
+    createMatrix(component, [chatMessageTypes])
+);
+
+export const tinyMessageSizing: StoryFn = createStory(html`
+    ${createMatrix(differentChatMessageSizeTestCase, [
+        [['a']],
+        [['outbound'], ['inbound'], ['system']]
+    ])}
+`);
+
+export const extraWideMessageSizing: StoryFn = createStory(html`
+    ${createMatrix(differentChatMessageSizeTestCase, [
+        [[loremIpsum]],
+        [['outbound'], ['inbound'], ['system']]
+    ])}
+`);
+
+export const manyLinesMessagSizing: StoryFn = createStory(html`
```

Comment:
```suggestion
export const manyLinesMessageSizing: StoryFn = createStory(html`
```
(note that this will change the name of the story so you'll have to re-approve the chromatic diffs)

Reply:
Fixed typo.  I will reapprove the chromatic diffs.